product_currency/models/product_template.py

```diff
@@ -2,9 +2,7 @@
 # For copyright and license notices, see __manifest__.py file in module root
 # directory
 ##############################################################################
-from odoo import models, fields
-from odoo.osv import fields as old_fields
-# from odoo.exceptions import ValidationError
+from odoo import models, fields, api
 import logging
 
 _logger = logging.getLogger(__name__)
```

Comment:
`_logger` is defined but is not used. please remove definition of `_logger` and also `import logging`